ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureRequest.java

```diff
@@ -217,6 +237,31 @@ public void setV1UiCustomization(@Nullable ThreeDSecureV1UiCustomization v1UiCus
         this.v1UiCustomization = v1UiCustomization;
     }
 
+    /**
+     * Optional. Sets all UI types that the device supports for displaying specific challenge user interfaces in the 3D Secure challenge.
+     * Possible Values:
+     * 01 BOTH
+     * 02 Native
+     * 03 HTML
+     *
+     * @param uiType {@link ThreeDSecureUiType} The UI type to request.
+     */
+    public void setUiType(@Nullable @ThreeDSecureUiType int uiType) {
+        this.uiType = uiType;
+    }
+
+    /**
+     * Optional. List of all the render types that the device supports for displaying specific challenge user interfaces within the 3D Secure challenge.
+     *
+     * When using `ThreeDSecureUIType.BOTH` or `ThreeDSecureUIType.HTML`, all `ThreeDSecureRenderType` options must be set.
+     * When using `ThreeDSecureUIType.NATIVE`, all `ThreeDSecureRenderType` options except `ThreeDSecureRenderType.RENDER_HTML` must be set.
+     *
+     * @param renderType specifies what render type to use in the 3D Secure challenge
+     */
+    public void setRenderType(ArrayList renderType) {
```

Comment:
Same comment re: specifying the default?

Reply:
done: https://github.com/braintree/braintree_android/pull/738/commits/7f414880a38aad233d2446ed6f1d4892a190d3b2